### DISCUSSION TITLE
제가 초대하지 않아도 Jira 보드를 볼 수 있나요?

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#6] [Q&A] [answered]
https://cpp-study.atlassian.net/jira/software/projects/CS/boards/3

Jira에 가입한 뒤 위 링크를 들어가서 한번 확인해주세요.

**Comment by Lcat12345** [comment, marked as answer]
### user... 에서 cpp-study.atlassian.net의 Jira에 액세스할 수 없습니다. 
권한이 없나봐요 다른 계정으로 시도하라네요.

**Reply by Lcat12345** [reply]
제 계정입니다.
Lcat
ljj3640@gmail.com

**Reply by Woon-2** [reply]
확인했습니다.
Jira 초대 링크 보냈습니다.
